raiden-dapp/src/store/index.ts

```diff
@@ -121,9 +130,14 @@ const store: StoreOptions<RootState> = {
         }
       );
     },
-    allTokens: (state: RootState): Token[] => {
-      return Object.values(state.tokens);
-    },
+    allTokens: (state: RootState): Token[] =>
+      Object.values(state.tokens).sort((a: Token, b: Token) => {
+        if (hasNonZeroBalance(a, b)) {
+          return a.balance! < b.balance! ? 1 : -1;
+        }
+
+        return a.symbol && b.symbol ? a.symbol.localeCompare(b.symbol) : 0;
```

Comment:
Sorting by symbol here as I think it's more exact than sorting by name? Happy to change it though